docs/references/account/update-email.md

```diff
@@ -1,2 +1,2 @@
-Update currently logged in user account email address. After changing user address, user confirmation status is being reset and a new confirmation mail is sent. For security measures, user password is required to complete this request.
+Update currently logged in user account email address. After changing user address, the user confirmation status will get reset. A new confirmation email is not sent automatically however and you will need to use the send confirmation email endpoint again. For security measures, user password is required to complete this request.
```

Comment:
Does the `however` make sense here?

Reply:
It would make a bit more sense to use `however you can` instead of `however you will` as however is normally used to explain contridiction of the previous point